asf_search/ASFProduct.py

```diff
@@ -232,6 +238,9 @@ def remotezip(self, session: ASFSession) -> 'RemoteZip':
 
         return remotezip(self.properties['url'], session=session)
 
+    def _has_multiple_files(self):
```

Comment:
Maybe `_has_additional_files` instead, so it lines up with the `additionalUrls` key it's checking? And it also implies they're "extra, maybe un-required" files more imo

Reply:
There was actually a regression by adding this method, instances where downloading ADDITIONAL_FILES would warn and ignore a user provided `fileName` even if `additionalUrls` had only 1 item. Removed it in favor of the original if-statement block.